include/odp/api/spec/traffic_mngr.h

```diff
@@ -248,6 +289,9 @@ typedef struct {
 	 * When true the min_weight and max_weight fields above specify
 	 * the legal range of such weights. */
 	odp_bool_t weights_supported;
+
+	/** Node statistics counter capabilities */
+	odp_tm_node_stats_capability_t stats;
```

Comment:
Could we potentially need other statistics capabilities at this level later on? In that case it could perhaps be better to name this `node_stats`.